src/native/minipal/types.h

```diff
@@ -4,6 +4,10 @@
 #ifndef HAVE_MINIPAL_TYPES_H
 #define HAVE_MINIPAL_TYPES_H
 
+#ifdef TARGET_WINDOWS
```

Comment:
```suggestion
#ifdef HOST_WINDOWS
```
This should be host. Other ifdefs in this file should probably be HOST_... too.

Reply:
I moved this macro from utf8.h as-is. After adding `#include <windows.h>`, it is giving us a different error (conflicting with windows.h definition). It's also used in mono-win, so I'd need to check how to fix it for the entire matrix. Alternatively, we can move it back to utf8.h since it's only a single type.